[2023-12-01T15:31:21.613+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: zillow_analytics_dag.tsk_is_file_in_s3_available manual__2023-12-01T15:29:37.526614+00:00 [queued]>
[2023-12-01T15:31:21.620+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: zillow_analytics_dag.tsk_is_file_in_s3_available manual__2023-12-01T15:29:37.526614+00:00 [queued]>
[2023-12-01T15:31:21.620+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 3
[2023-12-01T15:31:21.636+0000] {taskinstance.py:1382} INFO - Executing <Task(S3KeySensor): tsk_is_file_in_s3_available> on 2023-12-01 15:29:37.526614+00:00
[2023-12-01T15:31:21.640+0000] {standard_task_runner.py:57} INFO - Started process 36972 to run task
[2023-12-01T15:31:21.644+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'zillow_analytics_dag', 'tsk_is_file_in_s3_available', 'manual__2023-12-01T15:29:37.526614+00:00', '--job-id', '76', '--raw', '--subdir', '/home/ubuntu/airflow/dags', '--cfg-path', '/tmp/tmp4eou64ra']
[2023-12-01T15:31:21.645+0000] {standard_task_runner.py:85} INFO - Job 76: Subtask tsk_is_file_in_s3_available
[2023-12-01T15:31:21.679+0000] {task_command.py:416} INFO - Running <TaskInstance: zillow_analytics_dag.tsk_is_file_in_s3_available manual__2023-12-01T15:29:37.526614+00:00 [running]> on host ip-172-31-24-255.us-west-2.compute.internal
[2023-12-01T15:31:21.753+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='myemail@domain.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='zillow_analytics_dag' AIRFLOW_CTX_TASK_ID='tsk_is_file_in_s3_available' AIRFLOW_CTX_EXECUTION_DATE='2023-12-01T15:29:37.526614+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-12-01T15:29:37.526614+00:00'
[2023-12-01T15:31:21.753+0000] {s3.py:106} INFO - Poking for key : s3://myawsbucket-new/response_data_01122023153115.csv
[2023-12-01T15:31:21.758+0000] {base.py:73} INFO - Using connection ID 'aws_s3_conn' for task execution.
[2023-12-01T15:31:21.759+0000] {connection_wrapper.py:378} INFO - AWS Connection (conn_id='aws_s3_conn', conn_type='aws') credentials retrieved from login and password.
[2023-12-01T15:31:27.440+0000] {s3.py:106} INFO - Poking for key : s3://myawsbucket-new/response_data_01122023153115.csv
[2023-12-01T15:31:32.455+0000] {s3.py:106} INFO - Poking for key : s3://myawsbucket-new/response_data_01122023153115.csv
[2023-12-01T15:31:32.471+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-12-01T15:31:32.476+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=zillow_analytics_dag, task_id=tsk_is_file_in_s3_available, execution_date=20231201T152937, start_date=20231201T153121, end_date=20231201T153132
[2023-12-01T15:31:32.523+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-01T15:31:32.540+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
